WoWsShipBuilder.Web/Pages/Charts.razor

```diff
@@ -17,8 +16,8 @@
 
```

Comment:
This file has become way too big. Consider splitting or at least move C# code into separate codebehind file